pkg/postage/postagecontract/contract.go

```diff
@@ -207,6 +208,10 @@ func (c *postageContract) getBalance(ctx context.Context) (*big.Int, error) {
 }
 
 func (c *postageContract) CreateBatch(ctx context.Context, initialBalance *big.Int, depth uint8, immutable bool, label string) ([]byte, error) {
+	err := c.ExpireBatches(ctx)
+	if err != nil {
+		return nil, err
```

Comment:
this error should be wrapper with a meaningful error msg, like 
`fmt.Errorf("expire batches: %w", err)`

Reply:
and every else this is used

Reply:
so if we wrap it within ExpireBatches fn we probably do not need to wrap it anywhere else since the wrapped message will be returned nonetheless if i am correct.